src/Appwrite/Usage/Calculators/Database.php

```diff
@@ -95,6 +96,7 @@ protected function createOrUpdateMetric(string $projectId, string $metric, strin
                     'time' => $time,
                     'metric' => $metric,
                     'value' => $value,
+                    'region' => App::getEnv('_APP_REGION', 'default'),
```

Comment:
Will be better to pass this to the constructor so we don't couple the calculator with the class `App`